### ISSUE TITLE
Unable to install with `minishlink/web-push`

### ISSUE THREAD

**Issue by shishtpal** [googleapis/google-auth-library-php#274]
I am unable to install `google/apiclient:^2.0` because I have already installed `minishlink/web-push`

Please Update library to use latest version of `guzzlehttp/psr7` so I can use both libraries.

#### Environment details

  - OS: Windows
  - PHP version: 7.4

#### Command Output

```log
m:\rajkiran.local\rkic>php bin\composer.phar require google/apiclient:^2.0
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - google/apiclient v2.0.0 requires google/auth 0.8 -> satisfiable by google/auth[v0.8].
    - Installation request for google/apiclient 2.0 -> satisfiable by google/apiclient[v2.0.0].
    - Conclusion: remove guzzlehttp/psr7 1.6.1
    - Conclusion: don't install guzzlehttp/psr7 1.6.1
    - google/auth v0.8 requires guzzlehttp/psr7 1.2.* -> satisfiable by guzzlehttp/psr7[1.2.0, 1.2.1, 1.2.2, 1.2.3].
    - Can only install one of: guzzlehttp/psr7[1.2.0, 1.6.1].
    - Can only install one of: guzzlehttp/psr7[1.2.1, 1.6.1].
    - Can only install one of: guzzlehttp/psr7[1.2.2, 1.6.1].
    - Can only install one of: guzzlehttp/psr7[1.2.3, 1.6.1].
    - Installation request for guzzlehttp/psr7 (locked at 1.6.1) -> satisfiable by guzzlehttp/psr7[1.6.1].

```

Thanks!

**Comment by bshaffer** [comment]
This library does support the latest version of `guzzle/psr` (see [here](https://github.com/googleapis/google-auth-library-php/blob/master/composer.json#L12)), as `^1.2` means any version above `1.2` and below `2.0` ([composer docs](https://getcomposer.org/doc/articles/versions.md#caret-version-range-)).

The issue you have is you need to add `google/apiclient` to your `composer.json` manually and then run `composer update`. The error you're getting is because composer is trying to resolve `google/apiclient` to your locked dependencies and it is unable to do so.

**Comment by bshaffer** [comment]
It also seems like composer is trying to install `v2.0.0`, which is not the latest version. The latest version is `v2.4.1`. Judging by your CLI output, I don't know why this would happen, but you can fix it by using `google/apiclient:^2.4.1` instead.

**Comment by shishtpal** [comment]
Thanks 
It worked
